website/content/docs/enterprise/license/faq.mdx

```diff
@@ -7,35 +7,45 @@ description: >-
 
 # Frequently Asked Questions (FAQ)
 
-This FAQ is for the license changes introduced in Nomad Enterprise version v1.1.0+ent. 
-Nomad Enterprise automatically load Nomad licenses when a Nomad server agent starts using Nomad Enterprise.
+This FAQ is for the license changes introduced in Nomad Enterprise version v1.6.0+ent.
+
+For license-related FAQ for earlier versions, please refer to earlier versions of this document
+using the version drop-down on this page, e.g. [v1.5.x](/nomad/docs/v1.5.x/enterprise/license/faq).
 
 ## Q: Can I get a quick summary of the Nomad changes?
 
-Starting with Nomad Enterprise v1.1.0, the license enablement process is different.
+Starting with Nomad Enterprise v1.6.0, license expiration is different.
+
+For trial licenses, the behavior remains the same.  Enterprise features will stop working
+when the license expires, and a server can not start with an expired license.
```

Comment:
```suggestion
when the license expires, and a server cannot start with an expired license.
```

I think? How does English even work? 😅

Reply:
looks like grammarly agrees with you https://www.grammarly.com/blog/cannot-or-can-not/ but to me it's a little overly prescriptive.

I personally think they're interchangeable but that the extra space adds emphasis, but the important bit is the _"not"_ part, so as long as that's unambiguous I could roll with "cannot" 🤷 

Tangentially, in creative writing I tend to read "cannot" as a little, um, posh or hoity-toity?  I suppose because ordinary folks usually say "can't" unless they want to emphasize "can _not"_ which benefits from the space to indicate a short pause.  I could go extra informal like

> and a server _literally **can't even**_ start with an expired license.  🤣 🤣 🤣

Reply:
Ah interesting! Don't mind me then, I think I never thought about "can not" so it seemed a bit odd (can? not!) 😅